collections/Laud_Misc/MS_Laud_Misc_70.xml

```diff
@@ -61,25 +59,100 @@
                      <idno type="SCN">827</idno>
                   </altIdentifier>
                </msIdentifier>
-               <head>Jacobus de Voragine, <title>Sermones de sanctis</title>; Germany, 15th century</head>
+               <head>Pereginus de Oppeln (?), <title>Sermones de sanctis</title>; Germany, 15th century, second quarter or middle</head>
                <msContents>
+                  <textLang mainLang="la">Latin</textLang>
+                  <msItem>
+                     <note>Fol. ii recto blank except for <q>Assumpta est maria in celum</q> and, added, <q>1459</q>; fol. ii verso blank.</note>
+                  </msItem>
                   <msItem xml:id="MS_Laud_Misc_70-item1" n="1">
                      <locus>(fols. 1r-114r)</locus>
-                     <author key="person_100198644">Jacobus de Voragine</author>
-                     <title key="work_2331">Sermones de sanctis</title>
+                     <author key="person_63741243"><supplied>Peregrinus de Oppeln (?)</supplied></author>
```

Comment:
Interesting solution to this problem. Situations such as this make me wonder whether the catalogue entry should only concentrate on identifying the relevant works, leaving information about authorship to the authority files.